pkg/rhttp/datatx/manager/tus/tus.go

```diff
@@ -155,6 +160,10 @@ func (m *manager) Handler(fs storage.FS) (http.Handler, error) {
 		case "DELETE":
 			handler.DelFile(w, r)
 		case "GET":
+			metrics.DownloadsActive.Add(1)
+			defer func() {
+				metrics.DownloadsActive.Sub(1)
+			}()
```

Comment:
Have you tested this? I think this the wrong place for the download counter. Afair the `GetFile` method will only return files that are currently being uploaded.

Reply:
I tested this, and the download worked fine, it was the POST/PATCH that didn't work as expected, as the upload is chunked. Added the counter to the PATCH as well.